### PR TITLE
Fix broken link to remove tags from entries

### LINKED ISSUE #3534: Link to remove a tag from an entry seems broken
### Issue details

Links to remove a tag from an entry view seem broken and redirect to `/remove-tag//{tid}` –with {tid} the tag id– which lead to a 404 error.

### Environment

* wallabag version (or git revision) that exhibits the issue: _2.3.1_
* How did you install wallabag? Via `git clone` or by downloading the package? _git clone_
* Last wallabag version that did not exhibit the issue (if applicable): _2.3.0_
* php version: _7.0_
* OS: _Debian 9_
* type of hosting (shared or dedicated): _dedicated_
* which storage system you choose at install (SQLite, MySQL/MariaDB or PostgreSQL): _PostgreSQL_

### Steps to reproduce/test case

* Add a tag to an entry
* Remove the tag from the entry view
* Profit

### PR DESCRIPTION
**Pull request by Kdecherf**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | -
| Documentation | no
| Translation   | no
| CHANGELOG.md  | no
| Fixed tickets | #3534 
| License       | MIT

Fixes #3534

### CODE REVIEW

**Comment by Kdecherf** [comment]
@Simounet updated, thanks

**Comment by nicosomb** [comment]
Do you think that it's easy to add a test to avoid regressions again here?

**Comment by Kdecherf** [comment]
Done, there's a test now for this issue

File: tests/Wallabag/CoreBundle/Controller/EntryControllerTest.php [hunk 1]
```diff
@@ -1478,4 +1479,23 @@ public function testPostEntryWhenFetchFails()
...
+        $tag = $entry->getTags()[0];
+
+        $crawler = $client->request('GET', '/view/' . $entry->getId());
+
+        // As long as the deletion link of a tag is following
+        // a link to the tag view, we take the second one to retrieve
+        // the deletion link of the first tag
+        $link = $crawler->filter('body div#article div.tools ul.tags li.chip a')->extract('href')[1];
+
+        $this->assertSame(sprintf('/remove-tag/%s/%s', $entry->getId(), $tag->getId()), $link);
```
Comment (j0k3r) [inline tests/Wallabag/CoreBundle/Controller/EntryControllerTest.php]: Why don't you follow the link to ensure it returns a 301?

Comment (Kdecherf) [reply tests/Wallabag/CoreBundle/Controller/EntryControllerTest.php]: Do you want a complete test in the same PR or can we delay it to another PR?

Comment (j0k3r) [reply tests/Wallabag/CoreBundle/Controller/EntryControllerTest.php]: As you want 🎅

**Pull request merged**